pywbem/mof_compiler.py

```diff
@@ -2312,8 +2313,16 @@ def __init__(self, handle, search_paths=None, verbose=False,
             not depend on existing CIM elements in the repository.
 
           search_paths (:term:`py:iterable` of :term:`string`):
-            An iterable of path names of directories where MOF include files
-            should be looked up.
+            An iterable of path names of directories where compiler will search
+            for mof files to complete a compile operation.  The compiler
+            searches these paths (including subdirectories) for
+            mof in a number of cases including: 1) find a superclass that
+            is not in repository while compiling a class 2) Finding a
+            qualifier that is not in the repository (it looks for filenames
+            'qualifiers' and 'qualifiers_optional', 3) finding a dependent
+            class reference property and the EmbeddedInstanceQualifier)
+            Currently this works for superclasses but not for embedded
+            instance qualifier and reference properties/parameters.
```

Comment:
"Currently this works for superclasses" in item 3 created some confusion on my part until I realized that this part of the sentence is a statement about item 1. I suggest to change this sentence to only talk about item 3, i.e. change it to something like "(not currently implemented)".

Reply:
OK.